src/utils/daemon.rs

```diff
@@ -51,14 +51,10 @@ fn start_registry_watcher(opts: DocBuilderOptions, context: &dyn Context) -> Res
 
 pub fn start_daemon(context: &dyn Context, enable_registry_watcher: bool) -> Result<(), Error> {
     let config = context.config()?;
-    let dbopts = DocBuilderOptions::new(config.prefix.clone(), config.registry_index_path.clone());
-
-    // check paths once
-    dbopts.check_paths().unwrap();
```

Comment:
Hmm, shouldn't we keep this check in? What happens now if the path doesn't exist?

Reply:
The index fails to initialize.